2024/day01/day01.go

```diff
@@ -0,0 +1,47 @@
+/* Copyright © 2024 cpmachado <cpmachado@protonmail> */
+package day01
+
+import (
+	"log"
+	"math"
+	"slices"
+)
+
+// Represents a Location Id
+type LocationId int
```

Comment:
This looked reasonable at the time of the first time, maybe remove it in a refactor.